remake.yml

```diff
@@ -0,0 +1,15 @@
+sources:
+[D[D[D[D[D[D[D[D[D[D[B[A[C[C- code.R
```

Comment:
hmmm...wonder what happened here w/ the [D [D stuff?

Reply:
🤷 I just deleted it in the text editor. I think I might have hit tab? and then tried to delete the tab?

Reply:
perhaps. Did you use the `cat ...` instructions to create the file? I know that works in git bash, but unsure in windows `cmd`

Reply:
@jread-usgs I tried windows `cmd` first and that _did not_ work, so then I used git bash, following the instructions